Unity-Technologies/Graphics
# 2d/point light orientation fix

### Checklist for PR maker

---
### Purpose of this PR
This PR fixes an issue with 2D point lights that caused them to incorrectly render when used on a plane other than XY

---
### Testing status

**Manual Tests**: What did you do?
- [X ] C# and shader warnings (supress shader cache to see them)

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/902-Graphics
